lib/utils/index.test.ts

```diff
@@ -47,3 +47,9 @@ describe("isFunction", () => {
     expect(isFunction(input)).toEqual(expected);
   });
 });
+
+describe("createUUID", () => {
+  it("should return a string composed of valid hex characters or dashes `-`", () => {
+    expect(createUUID()).toMatch(/^[0-9a-f\-]+$/);
```

Comment:
No need to test this more. We don't actually care that this is a valid uuid. What's important is that:
- it's random
- it contains only hex characters so that they don't get encoded when used in a cookie.